sympy/polys/matrices/sdm.py

```diff
@@ -901,8 +903,12 @@ def lu_solve(A, b):
 
     def nullspace(A):
         """
+        Nullspace of a :py:class:`~.SDM` matrix A.
```

Comment:
This SDM (or DDM) method is no longer used by `DomainMatrix.nullspace` because it now uses `nullspace_from_rref` in combination with either `rref` or `rref_den` which makes it easier to control what is happening from a higher-level.